modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/MissingRequiredFieldsPopover.es.js

```diff
@@ -20,6 +20,7 @@ export default function MissingRequiredFieldsPopover({
 	message = {},
 	nativeField,
 	onClick,
+	popoverSide = 'left',
```

Comment:
alignPopoverPosition is better, I guess. What you guys think?

Reply:
> alignPopoverPosition is better, I guess. What you guys think?

I think the word Popover is a little redundant in this case, since it's a prop inside the popover component itself. alignPosition would be sufficient for me :)

Reply:
Nice. I'll fix this

Reply:
Thx @ramonmsp